include/gevolution/output.hpp

```diff
@@ -147,6 +147,27 @@ void writeLightcones (
     Field<Cplx> *BiFT, Field<Cplx> *SijFT, PlanFFT<Cplx> *plan_Bi,
     PlanFFT<Cplx> *plan_Sij, int &done_hij, std::set<long> *IDbacklog);
 
+void writeLightcone_shell(
+    const std::string filename,
+    const gadget2_header hdr,
+    const lightcone_geometry lc,
+    const Particles_gevolution &Pcdm)
+// Notice that lc describes the lightcone shell we want to save. The caller of this function is the
+// one responsible to split a full lightcone into thin lightcone shells as well as construct the
+// snapshot header template.
+{
+    auto particles_in_lightcone = [&](const gevolution::particle&, const LATfield2::Site&)
+    {
+        // FIXME: select the particles that lie in the lightcone
```

Comment:
This is important!

Reply:
Not important. This was part of a previous I had and I didn't go forward. Now the particle selection and lightcone serialization is taken care of by the `parallel_lightcone` data structure.